obelisk/python/obelisk_py/zoo/robot/hardware/leap_hand/leap_node.py

```diff
@@ -0,0 +1,67 @@
+import os
+import sys
+import math
+
+file_dir = os.path.dirname(__file__)
+sys.path.append(file_dir)
+
+import dxl_motor_helper as dxl
+from dynamixel_sdk import *
+
+from obelisk_py.core.robot import ObeliskRobot
+from obelisk_py.core.obelisk_typing import ObeliskControlMsg, ObeliskSensorMsg
+from obelisk_sensor_msgs.msg import JointEncoders
+
+
+
+class ObeliskLeapHand(ObeliskRobot):
+
+    N_MOTORS = 16
+
+    def __init__(self, node_name):
```

Comment:
i'm gonna put these comments here in a centralized place, but you need docstrings on all public classes and methods of public classes as well as public methods. Also, i'm requesting that you use python typing conventions for all arguments and make sure that python type checking passes to the best of your ability. You can check this locally by calling `pyright` in the repo root, and it will tell you what isn't passing (warnings are fine, errors are not).

there might be some things that are very stubborn in terms of the type checking - feel free to ping me on slack to ask about how to resolve. for the truly unresolvable, we can always annotate and make `pyright` ignore.